lessons/Data-Science/measures-of-center/langs/en-us/pages/histograms-moc.adoc

```diff
@@ -0,0 +1,77 @@
+= Histograms and Measures of Center
+
+@n The two histograms below represent the amount of money spent on a pair of jeans, one for a sample of sixth grade students and one for a sample of eighth grade students.
+
+[cols="^1a,^1a"]
+|===
+
+| @image{../images/jeans-cost-6th.png, 275}  | @image{../images/jeans-cost-8th.png, 275}
```

Comment:
All of the images on this page have a TON of whitespace around them. If you trim this off, there's a lot more space for students to write their answers. The current space provided feels too small to be useful.